media_driver/agnostic/gen11/codec/kernel_free/media_srcs.cmake

```diff
@@ -18,6 +18,33 @@
 # ARISING FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR
 # OTHER DEALINGS IN THE SOFTWARE.
 
+if (BUILD_KERNELS)
+
+    function(gen_cm_kernel_from_source name platform)
```

Comment:
Please, move this function definition to https://github.com/intel/media-driver/blob/master/media_driver/agnostic/media_srcs.cmake